push/templates/push/validation.html

```diff
@@ -9,18 +9,22 @@
 {% endblock %}
 
 {% block content %}
-    <p>To verify you own the signing key for your app, please sign the following token using the same key you use to sign this application's VAPID JWT, and paste the url-safe, base64-encoded value below.</p>
-    <dl>
-        <dt>Token:</dt>
-        <dd>{{ app.vapid_key_token }}</dd>
-    </dl>
+    {% if app.vapid_key_status != "recording" %}
```

Comment:
Since `valid` is another `vapid_key_status` value, I made `app.valid` as a method to check if the status is either `valid` *OR* `recording`. So, this should be:
```
{% if not app.valid %}
```